### PR TITLE
Yaml document related changes code is working fine

### PR DESCRIPTION
**Pull request by md-rafeek**
## Description
[CSCwm83940] - Updating Power Level via Ansible is unsuccessful
[CSCwm47563] - Update documentation to change secondary, tertiary controller names as mandatory field

## Type of Change
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [x] Documentation update

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] All the sanity checks have been completed and the sanity test cases have been executed

## Ansible Best Practices
- [ ] Tasks are idempotent (can be run multiple times without changing state)
- [ ] Variables and secrets are handled securely (e.g., using `ansible-vault` or environment variables)
- [x] Playbooks are modular and reusable
- [x] Handlers are used for actions that need to run on change

## Documentation
- [x] All options and parameters are documented clearly.
- [x] Examples are provided and tested.
- [x] Notes and limitations are clearly stated.

### CODE REVIEW

File: plugins/modules/accesspoint_workflow_manager.py [hunk 1]
```diff
@@ -263,7 +263,7 @@
             description: Mode of power assignment for the 2.4GHz radio interface. Accepts "Global" or "Custom". For example, "Custom".
             type: str
             required: False
-          power_level:
```
Comment (madhansansel) [inline plugins/modules/accesspoint_workflow_manager.py:266]: Do we need to change here or in the code?

Comment (md-rafeek) [reply plugins/modules/accesspoint_workflow_manager.py:266]: In the code i have used powerlevel, hence the api response of AP config also same. i have fixed the code long back forget to change in this document.

**Review by md-rafeek — COMMENTED** [review]
addressed the commnets.

File: plugins/modules/accesspoint_workflow_manager.py (hunk 1 above)
Comment (madhansansel) [reply plugins/modules/accesspoint_workflow_manager.py:255]: If power_level is released to customer, then we can use power_level itself. 
power_level is more consistent with YAML's preference for clarity and common conventions.

Why did you change power_level into powerlevel in the code?

Comment (md-rafeek) [reply plugins/modules/accesspoint_workflow_manager.py:255]: When we merged first time we used the power_level QA bug was raised not working in power level. then we checked the code. while checking we found update ap config the payload was changed to powerLevel.  actually in the payload as well as ap config response, both powerlevel only using. we are using function to convert snake case to camel case to create payload. there we are getting problem. those we fixed in the last bug with powerlevel. that i need to update in the yml document i missed it

**Review by md-rafeek — COMMENTED** [review]
if we change to power_level, the dynamic function used for snake_case to camel case will have problem. so we used powerlevel

**Review by madhansansel — APPROVED** [review]

**Pull request merged**